docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/button-click/index.md

```diff
@@ -0,0 +1,339 @@
+| `sp.js`              | ❌        |
+| `sp.lite.js`         | ❌        |
+
+**Download:**
+
+<table>
+  <tbody>
+    <tr>
+      <td>Download from GitHub Releases (Recommended)</td>
+      <td>
+        <a
+          href="https://github.com/snowplow/snowplow-javascript-tracker/releases"
+          >Github Releases (plugins.umd.zip)</a
+        >
+      </td>
+    </tr>
+    <tr>
+      <td>Available on jsDelivr</td>
+      <td>
+        <a
+          href="https://cdn.jsdelivr.net/npm/@snowplow/browser-plugin-button-click-tracking@latest/dist/index.umd.min.js"
+          >jsDelivr</a
+        >
+        (latest)
+      </td>
+    </tr>
+    <tr>
+      <td>Available on unpkg</td>
+      <td>
+        <a
+          href="https://unpkg.com/@snowplow/browser-plugin-button-click-tracking@latest/dist/index.umd.min.js"
+          >unpkg</a
+        >
+        (latest)
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+</TabItem>
+<TabItem value="browser" label="Browser (npm)">
+
+- `npm install @snowplow/browser-plugin-button-click-tracking`
+- `yarn add @snowplow/browser-plugin-button-click-tracking`
+- `pnpm add @snowplow/browser-plugin-button-click-tracking`
+
+</TabItem>
+</Tabs>
+
+## Initialization
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+window.snowplow(
+    'addPlugin',
+    'https://cdn.jsdelivr.net/npm/@snowplow/browser-plugin-button-click-tracking@latest/dist/index.umd.min.js',
+    ['buttonClickTracking', 'SnowplowButtonClickTrackingPlugin']
+);
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+import { ButtonClickTrackingPlugin } from '@snowplow/browser-plugin-button-click-tracking';
+
+newTracker('sp1', '{{collector_url}}', { 
+   appId: 'my-app-id', 
+   plugins: [ ButtonClickTrackingPlugin() ],
+});
+```
+
+  </TabItem>
+</Tabs>
+
+## Overview
+
+To start tracking all button clicks, call the `enableButtonClickTracking` function:
+
+<Tabs groupId="platform" queryString>
+<TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+window.snowplow('enableButtonClickTracking');
+```
+
+</TabItem>
+<TabItem value="browser" label="Browser (npm)">
+
+```javascript
+import { enableButtonClickTracking } from '@snowplow/browser-plugin-button-click-tracking';
+enableButtonClickTracking();
+```
+
+</TabItem>
+</Tabs>
+
+Then, to stop tracking button clicks, call the `disableButtonClickTracking` function:
+
+<Tabs groupId="platform" queryString>
+<TabItem value="js" label="JavaScript (tag)" default>
+
+```javascript
+window.snowplow('disableButtonClickTracking');
+```
+
+</TabItem>
+<TabItem value="browser" label="Browser (npm)">
+
+```javascript
+import { disableButtonClickTracking } from '@snowplow/browser-plugin-button-click-tracking';
+disableButtonClickTracking();
+```
+
+</TabItem>
+</Tabs>
+
+## Configuration
+
+### Choosing which buttons to track
+
+There are three ways to configure which buttons are tracked:
```

Comment:
You could say there are four ways to configure – one being without configuration. I think it's worth mentioning what is tracked when no configuration is provided.